package.json

```diff
@@ -30,6 +30,7 @@
   "homepage": "https://github.com/twskj/pretty-swag#readme",
   "dependencies": {
     "esprima": "^4.0.0",
+    "expand-template": "^2.0.3",
```

Comment:
You might not need it. Modern js can do template out of the box by using backtick
```
var name = "world";
console.log(`hello ${name}`);
```

Reply:
I am using it for the method you have posted here. If you see the [OpenAPI specification for server object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#serverObject), you can see that user can add place holder to the URL and provide possible values for that.

Reply:
I am using it here: https://github.com/twskj/pretty-swag/pull/62/files#diff-44371df60e97437c264594912e8623a7R868